application/src/test/java/org/thingsboard/server/transport/lwm2m/client/LwM2mBinaryAppDataContainer.java

```diff
@@ -81,8 +81,11 @@ public LwM2mBinaryAppDataContainer() {
     public LwM2mBinaryAppDataContainer(ScheduledExecutorService executorService, Integer id) {
         try {
             if (id != null) this.setId(id);
-            executorService.scheduleWithFixedDelay(() ->
-                    fireResourcesChange(0, 2), 1800000, 1800000, TimeUnit.MILLISECONDS); // 30 MIN
+            executorService.scheduleWithFixedDelay(() -> {
+                        fireResourceChange(0);
+                        fireResourceChange(2);
+                    }
+                    , 1800000, 1800000, TimeUnit.MILLISECONDS); // 30 MIN
```

Comment:
fireResourcesChange(getResourcePath(0), getResourcePath(2)), 1800000, 1800000, TimeUnit.MILLISECONDS)

Reply:
it`s the same

Reply:
`... , 30, 30, TimeUnit.MINUTES)`